Bugs/68621 selected facets overlap filters mobile

a-rousseau opened the pull request:
[AB#68621](https://dev.azure.com/orckestra001/da965986-c85b-43be-9915-219568b000e5/_workitems/edit/68621)
[AB#68777](https://dev.azure.com/orckestra001/da965986-c85b-43be-9915-219568b000e5/_workitems/edit/68777)
Fix layout and CSS for a nice sticky layout on desktop
Copied and adaptted the dropdown buttons to select filters and Sorting options for the mobile view, to the div id leftCol
Added access to SelectedSortBy and AvailableSortBys parameters to raor functions
Added those parameters to the vue component as well
Fixed some styling issues when scrolling with - allow to see the search title with result count
Fix whitespace on desktop to allow View Filter button and SortBy button to be on the same line

Commit:
Fix sticky position styling

Fix layout and CSS for a nice sticky layout on desktop

Commit:
Merge branch 'dev' into bugs/68621-selected-facets-overlap-filters-mobile

Commit:
Copy Filter and SortBy dropdowns to the left column for mobile

Copied and adaptted the dropdown buttons to select filters and Sorting options for the mobile view, to the div id leftCol
Added access to SelectedSortBy and AvailableSortBys parameters to raor functions
Added those parameters to the vue component as well
Fixed some styling issues when scrolling with - allow to see the search title with result count
Fix whitespace on desktop to allow View Filter button and SortBy button to be on the same line

InnaBoitsun commented on src/Orckestra.Composer.Website/App_Data/Razor/Composer/Search/BaseSelectedFacets.cshtml:
```diff
@@ -15,6 +15,29 @@
+    public ProductSearchResultsViewModel ProductSearchVM { get; set; }
+
+    
+    public SelectedSortBy SelectedSortBy { 
```
please also validate Query/Set pages with this change

Commit:
Apply Facets updates to Queries

Pass missing data to BaseSelectedFacets from Merchandising and Product Set Queries
Remove bg-light classes from facets cards
